app/Models/Login.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
```

Comment:
For consistency it'd be great to extend from `App\Models\Model` vs the base laravel one.
I'd also be tempted to move this under `App\Models\User\Login` rather than in the Models namespace. (Just to be consistent with the other sub resources).

Reply:
Ok i will change it.